CSC8503CoreClasses/UI.cpp

```diff
@@ -85,6 +85,7 @@ void UI::BuildVerticesForIcon(const Vector2& iconPos, int horiSize, int vertSize
 }
 
 void NCL::CSC8503::UI::ChangeEquipmentSlotTexture(int slotNum, Texture& texture) {
+	return;
```

Comment:
I personally don't understand why return directly, Might be use it later not for now?